tests/test_api_v1.py

```diff
@@ -1707,7 +1706,11 @@ def test_request_patch_invalid(
 
     rv = client.patch(f"/api/v1/requests/{request_id}", json=payload, environ_base=worker_auth_env)
     assert rv.status_code == status_code
-    assert rv.json == {"error": message}
+
+    if "detail" in rv.json:
+        assert message in rv.json["detail"]
+    else:
+        assert message in rv.json["error"]
```

Comment:
Did half of our error messages move to "detail" while the other half stayed in "error"? That might be on the not-OK side of breaking changes

Reply:
^ would break OSBS https://github.com/containerbuildsystem/atomic-reactor/blob/10dce319b239a2bb93cdd78c4506fb2fa7103b80/atomic_reactor/utils/cachito.py#L97

Reply:
I remember finding a way to override the format of connexion's error response and return ValidationError just like we do today. :thinking:

Reply:
I think the solution was in creating a [custom validator](https://connexion.readthedocs.io/en/latest/request.html#custom-validators)